CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+How to contribute?
+
+Basic flow:
+- fork this repository
+- setup the project: `npm install`
+- start dev mode `npm run dev` and open http://localhost:8090/__zuul
```

Comment:
Well maybe we should remove this line or add a step with the following statement : "make your contribution"?